README.md

```diff
@@ -101,3 +101,12 @@ To run the tests, `libvirt` must be installed and the daemon running:
 zypper in libvirt
 systemctl start libvirtd
 ```
+
+### Known issues
+* Failed to run [wait-for-total-initialization](./cluster/join.sls):
```

Comment:
I would change that description to:
```
Failure running [join-the-cluster](./cluster/join.sls):
> Note: The current solution is to query **hawk** status. This doesn't exactly
> represent the state of **pacemaker** (hawk initialization is independent).
> Increasing the `join_timer` to wait pacemaker may help in some cases.
```

Reply:
Thanks, really helpful.
 English is always an issue to me, especially to write a decent description. sigh.